index.js

```diff
@@ -114,7 +114,9 @@ function addAll(packages, packageIndex) {
 		
 	// iterate over packages and prepare urls before I call the registry
 	for (var p in packages) {
-
+		if(p.indexOf("@") == 0) {
```

Comment:
I never use `==` in my code.
Also I'm not sure I understand, you're checking if the package name starts with `@` ?